particles/Tutorial/Kotlin/Demo/README.md

```diff
@@ -0,0 +1,89 @@
+# Behold Tic Tac Toe
+
+## Articles about Designing with Particles
+
+So from our introductory tutorials, we have the following definition of a particle:
+
+> *Particle* - Modular component of functionality. Ideally small units so particles can be reusable. 
+
+While that tells us _what_ a particle is, it does not say _how_ a particle should be used and _how_
+a recipe should be divided into particles. To better understand how to design a system in Arcs, it
+is important to have a full mental picture of particles. Just as there are multiple sides to any 
+person, particles have multiple faces. To fully understand them, we must look below the surface to
+see the soul of particles.
+
+#### Smallest Components
+Even from the definition of particles, we know they should be small components. Just as with object
+oriented programming, this lets us have easily interchangeable components. One can envision a world
+where particles are made by external developers and combined to form recipes - much the way 
+developers use external libraries in object-oriented languages.
+
+However, making particles as small as possible leads us towards a "turtles all the way down"
+mentality. There is obvious overhead involved in each particle, therefore the desire for small
+particles must be balanced against having too many.
+
+#### Bounds of Data Knowledge
+Arcs' mission statement says that users have sovereignty over their data and its use. To ensure
+data only travels as intended, Arcs uses data flow analysis to make grantees about data flow. But,
+for this to work properly, particles must delineate knowledge boundaries. Just as each nation has a 
+border so customs and immigration can know where to operate, we need to create systems with well 
+defined boundaries so Arcs can know where data should (and more importantly, shouldn't) exist.
+
+
+#### Functional Components
+When we create recipes, there are going to be occasions when it makes sense to extract some 
+functionality its own particle. This is easiest to understand with an example, which we provide in
+more detail below.
+
+### Tic-Tac-Toe, here we go!
+
+Alright, so let's get to putting this in practice with our tic-tac-toe game. As with any good
+engineering project, we need a set of requirements. Here are ours:
+ 1. Play Tic Tac Toe with the traditional rules.
+ 2. Be able to have a human play against a computer.
+ 3. The players (human or computer) may make invalid moves at invalid times.
+ 4. The main UI should not be trusted with any personal information, such as names.
```

Comment:
This one seems a little odd (doesn't the UI need to display the name?).

Could you rephrase it a little? Something like "Personal information, such as names, should be restricted to the components that need to know about them"

Reply:
Presumably this is for an anonymous game of tic tac toe (the stakes may be high and we don't want people to hold a grudge :P )

Reply:
The use case @shaper and I joke about is that maybe you're playing during work hours, so you don't want your boss to know....